presentation/common/src/presentation-common/EC.ts

```diff
@@ -179,6 +179,17 @@ export namespace PropertyInfo {
     return { ...info, classInfo: ClassInfo.toJSON(info.classInfo) };
   }
 
+  /** Serialize [[PropertyInfoJSON]] to compressed JSON */
+  export function toCompressedJSON(json: PropertyInfoJSON, classesMap: { [id: string]: CompressedClassInfoJSON }): PropertyInfoJSON<string> {
```

Comment:
shouldn't this take `PropertyInfo` instead of `PropertyInfoJSON`?

Reply:
Yep, it should. fixed it